model/import/AssemblerService.php

```diff
@@ -47,49 +48,77 @@
 class AssemblerService extends ConfigurableService implements AssemblerServiceInterface
 {
     use LoggerAwareTrait;
+    use OntologyAwareTrait;
 
     const MANIFEST_FILE = 'manifest.json';
     
     const RDF_FILE = 'delivery.rdf';
 
     const OPTION_FILESYSTEM_ID = 'filesystemId';
 
+    /**
+     * @var string Path to assembly import folder.
+     */
+    private $importFolder;
```

Comment:
why add a state to a stateless service? What is the advantage that outweighs the risk?

Reply:
It was added to do not pass this parameter between in all internal calls within this class